### PR TITLE
add Getting Started to README.md

### PR DESCRIPTION
**Pull request by skatkov**
Hey.

As all other lazy developers i was looking for a good lib for kraken bitcoin exchange. And yours seems much better then popular choices.

This is not a big contribution, but who knows - maybe will increase usage  ;-)

### CODE REVIEW

**Comment by jonatack** [comment]
Hi @skatkov, thank you for your PR. Somehow I didn't see it this whole time. Please accept my apologies. I will be working some more on this project soon and will look at your ideas. Thanks again.

**Comment by jonatack** [comment]
This library isn't published yet as a gem. Correct me if I am wrong, but I don't think these instructions will work until it is published.

File: README.md [hunk 1]
```diff
@@ -30,6 +30,23 @@ Currently developed with Ruby 2.4.0. Written for Ruby 2.3 and up.
...
 
+Add to your Gemfile
+```
+gem 'kraken_ruby_client', github: 'jonatack/kraken_ruby_client'
+```
+
+Install
+
+```
+bundle isntall
```
Comment (sirwolfgang) [inline README.md]:
```suggestion
bundle install
```

**Comment by jonatack** [comment]
Thanks, everyone. Done with https://github.com/jonatack/kraken_ruby_client/pull/13. @skatkov I moved your commit there and built the changes from it.

File: README.md (hunk 1 above)
Comment (jonatack) [reply README.md]: Thanks @sirwolfgang for reviewing. Change done in https://github.com/jonatack/kraken_ruby_client/pull/13.

**Pull request closed without merging**